mysql/README.md

```diff
@@ -1,59 +1,131 @@
 # Overview
 
-Connect MySQL to Datadog in order to:
+The Datadog agent's MySQL check sends many database metrics to Datadog, including metrics for:
 
-* Visualize your database performance
-* Correlate the performance of MySQL with the rest of your applications
+* Query throughput
+* Query performance (average query run time, slow queries, etc)
+* Connections (currently open connections, aborted connections, errors, etc)
+* InnoDB (buffer pool metrics, etc)
 
-# Installation
+And [many more](https://github.com/DataDog/integrations-core/blob/master/mysql/metadata.csv). You can also invent your own metrics using custom queries.
```

Comment:
Another option: use Markdown comments to mark beginning and end of snips that Docs site should pull down rather than literally include.